### ISSUE TITLE
Incorrect error "Module xxx is private" with include!("../../examples/xxx/mod.rs") in integration tests

### ISSUE THREAD

**Issue by qiujiangkun** [intellij-rust/intellij-rust#7133]
I have a project like this
```
examples
├── logger
│   └── mod.rs -- fn hello(){}
└── some_example.rs 
tests
├── logger
│   └── mod.rs -- include!("../../examples/logger/mod.rs")
└── some_test.rs -- nothing
src
└── lib.rs -- nothing
```
some_example.rs
```rust
mod logger;

fn main() {
    logger::hello();
}
```
![image](https://user-images.githubusercontent.com/33482468/115116008-fbff7d80-9fc9-11eb-8860-ab4ad310c4da.png)

[example.zip](https://github.com/intellij-rust/intellij-rust/files/6329376/example.zip)
